internal/pkg/develop/plugin/create.go

```diff
@@ -0,0 +1,42 @@
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// Create creates a new plugin.
+// 1. Render template files
+// 2. Persist all files
+// 3. Print help information
+func Create() error {
+	name := viper.GetString("name")
+	if name == "" {
+		return fmt.Errorf("the name must be not \"\", you can specify it by --name flag")
+	}
+	log.Debugf("Got the name: %s.", name)
+
+	p := NewPlugin(name)
+
+	// 1. Render template files
+	files, err := p.RenderTplFiles()
```

Comment:
It seems "RenderTplFiles" comes from the same package. Do we have to export it? Same for "PersistFiles".

Reply:
Yes, these 2 functions are only used in the same package now.  But I still choose to `Export` it for these 2 are the main functions at the `Create()` logic. I think it's more clear to write:
```go
func (p *Plugin) RenderTplFiles(){}
func (p *Plugin) renderTplFile(){}
```

compare to

```go
func (p *Plugin) renderTplFiles(){}
func (p *Plugin) renderTplFile(){}
```